apps/src/ImageSegmentationMain.cpp

```diff
@@ -0,0 +1,29 @@
+#include "rt/DisegniSegmenter.hpp"
+
+using namespace cv;
+
+int main(int argc, char* argv[])
+{
+
+    // TESTING PURPOSE
+    cv::Mat input =
+        cv::imread(samples::findFile("DesigniTest2.png"), cv::IMREAD_COLOR);
+
+    // Checks for valid input
+    if (input.empty()) {
+        std::cout << "Could not open or find the image" << std::endl;
+        return EXIT_FAILURE;
+    }
+    // TESTING PURPOSE
+
+    DisegniSegmenter a;
```

Comment:
It's currently easy to keep track of what variable `a` is, but that won't necessarily always be the case. Change `a` to `segmenter` or some other descriptive variable name.